hieradata/class/production/jenkins.yaml

```diff
@@ -55,8 +55,8 @@ govuk_jenkins::job_builder::jobs:
   - govuk_jenkins::jobs::signon_cron_rake_tasks
   - govuk_jenkins::jobs::smokey
   - govuk_jenkins::jobs::smokey_deploy
-  - govuk_jenkins::jobs::transition_load_all_data
-  - govuk_jenkins::jobs::transition_load_site_config
+  - govuk_jenkins::jobs::transition_load_all_data_redirect
```

Comment:
We should not do this in production just yet, as the jobs are currently still needed (see above)